src/common/lib/types/protocolmessage.ts

```diff
@@ -23,6 +23,7 @@ const actions = {
   MESSAGE: 15,
   SYNC: 16,
   AUTH: 17,
+  ACTIVATE: 18,
```

Comment:
This is not documented in the draft

Reply:
This is only used for upgrade (no other SDKs will need this) - we haven't agreed what the message will be called so I just added this for the draft